academic_affairs.sql

```diff
@@ -80,6 +92,8 @@ SELECT
   GD.site_short,
   SUM(above_325_gpa) AS aa_above_325_gpa,
   SUM(composite_ready) AS aa_composite_ready,
```

Comment:
@jbakerr Similar question for rows 95-96 as on row 25

Reply:
same answer / same kpi as pointed out above. 

    -- % of seniors who are composite ready

Reply:
sorry, yeah same KPI % of not ready and near ready students who become composite ready by the highest official exam score